lib/static_sync/processor.rb

```diff
@@ -65,6 +52,35 @@ def local_filtered_files
 
     private
 
+    def look_for_conflicts(file)
+      if @storage.has_file?(file.version)
+        handle_conflict(file)
+      else
+        log.info("Uploading #{ file }") if @config.log
```

Comment:
Good point. Although, this is intended to be a quick fix. TBH, I'd almost rewrite the whole file